api/src/main/java/org/openmrs/module/ugandaemrreports/library/Moh105CohortLibrary.java

```diff
@@ -555,8 +559,8 @@ public CohortDefinition testedHivNegative() {
      * Ever Tested for HIV Before
      * @return CohortDefinition
      */
-    public CohortDefinition everTestedForHivBefore() {
-        return definitionLibrary.hasObs(Dictionary.getConcept(Metadata.Concept.HIV_TEST), Dictionary.getConcept(Metadata.Concept.YES_CIEL));
+    public CohortDefinition individualsTestingFortheFirstTime() {
```

Comment:
@solemabrothers Why delete the old definition which is used in previous reports? They are now broken